superset-frontend/src/constants.ts

```diff
@@ -111,6 +111,14 @@ export const URL_PARAMS = {
     name: 'focused_chart',
     type: 'number',
   },
+  language: {
```

Comment:
I believe the correct parameter here should be locale which is comprised of both the language and region. Another point is that we should be able to infer the currency from the locale.